client/state/utils/reducer.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import { UTILS_UNIQUEID_INCREMENT } from 'state/action-types';
+
+export function uniqueId( state = 1, action ) {
+	switch ( action.type ) {
+		case UTILS_UNIQUEID_INCREMENT:
+			state = state + 1;
+			break;
+	}
+
+	return state;
+}
+
+export default combineReducers( {
+	uniqueId
+} );
```

Comment:
We don't really seem to gain much here using `combineReducers()` except maybe for the fact that it automatically sets the key name. I can easily imagine setting up the code like this if we think we'll extend the reducers in the future, but that seems unlikely for `uniqueId`

I'm cool leaving it in, but uneasy about doing it when not needed/not adding value.